src/components/Profile/Profile.css

```diff
@@ -1,6 +1,8 @@
 .profileContainer {
   display: flex;
-  font-size: 100%;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
```

Comment:
good job giving the tables the flex display, but I suggest you give a justify-content of space around, and an align-items value of flex-start, so that the tables look uniform.